### DISCUSSION TITLE
mounting /var/log/journal persistently with systemd enabled

### DISCUSSION THREAD

**Discussion by jwcard** [containers/podman#24817] [Q&A] [answered]
I have several containers running and would like to mount each of their `/var/log/journal` directories on the host file system in a tree similar to `/var/log/container/$CONTAINER-NAME/journal`

According to the documentation, enabling systemd (which I do need) causes `/var/log/journal` to be mounted via `tmpfs`. Is there a way to get around this so that /var/log/journal uses persistent mode so that I can have it mounted in the hosts system?

**Comment by Luap99** [comment, marked as answer]
Did you test this? Generally all the internal podman mounting checks if the user already specified a mount themselves and then not overwrite the user mounts.

**Reply by jwcard** [reply]
I did now! I tried pretty much everything I could think of EXCEPT this. I had originally mounted _**/var/log**_ from inside the container and was trying to mount the tree with one bind mount. I tried adjusting permissions outside, configuration of journal etc.

Thank you so much for your answer. Mounting _**/var/log/journal**_ from inside the container to my host file system solved the issue.
